cli/command_snapshot_create.go

```diff
@@ -286,6 +286,19 @@ func (c *commandSnapshotCreate) snapshotSingleSource(ctx context.Context, fsEntr
 	manifest.Tags = tags
 	manifest.UpdatePins(c.pins, nil)
 
+	progress := c.svc.getProgress()
+
+	manifest.Progress = map[string]int64{
```

Comment:
I've listed all integer fields of `cliProgress` here, except `inProgressHashing` because it is "in progress" and thus not interesting to expose to the json